client/src/components/LogInForm.js

```diff
@@ -1,57 +1,110 @@
-            id="password"
-            className="form-control mb-2"
-            placeholder="Password"
-            required={true}
-            autoComplete="off"
-            tabIndex="3"
-            aria-invalid="false"
-            aria-controls="password"
-            aria-label="Choose a strong password."
-            aria-activedescendant="password-item-0"
-          />
-        </div>
-        <div>
-          <input
-            type="submit"
-            value="Log In"
-            tabIndex="4"
-            className="btn btn-lg btn-black btn-subscribe btn-block"
-          />
-        </div>
-      </form>
-    );
+function LogInForm({ history }) {
+  const [loading, setLoading] = useState();
+  const dispatch = useDispatch();
+  const { values, handleChange, handleSubmit } = useForm(
+    { email: '', password: ''},
+    postLogIn
+  );
+
+  async function postLogIn() {
+    try {
+      setLoading(true);
+
+      const { data } = await axios.post(
+        `${process.env.REACT_APP_EXPRESS_SERVER}/login`,
+        values
+      );
+      
+      const res = await data;
+
+      if (res.error) {
+        console.error(res.error);
+        return history.push('/signup-error');  
+      }
+
+      if (res.token) {
+        dispatch({
+          type: actions.SET_AUTH_USER,
+          authUser: {
+            email: values.email
+          }
+        });
+  
+        localStorage.setItem('token', res.token);
+  
+        setLoading(false);
+        return history.push('/');
+      }
+    } catch (error) {
+      setLoading(false);
+      console.error(error);
+    }
   }
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <div className="form-group">
+        <label htmlFor="email" className="d-none">
+          Email address
+        </label>
+        <input
+          onChange={handleChange}
+          value={values.email}
+          name="email"
+          type="email"
+          id="email"
+          className="form-control mb-2"
+          placeholder="Email address"
+          required
+          autoComplete="off"
+          tabIndex="2"
+          autoCapitalize="none"
+          aria-invalid="false"
+          aria-controls="email"
+          aria-label="What is your email address?"
+          aria-activedescendant="email-item-0"
+        />
+        <label htmlFor="password" className="d-none">
+          Password
+        </label>
+        <input
+          onChange={handleChange}
+          value={values.password}
+          name="password"
+          type="password"
+          id="password"
+          className="form-control mb-2"
+          placeholder="Password"
+          required
+          autoComplete="off"
+          tabIndex="3"
+          aria-invalid="false"
+          aria-controls="password"
+          aria-label="Choose a strong password."
+          aria-activedescendant="password-item-0"
+        />
+        <label htmlFor="passphrase" className="d-none">
+          Passphrase
```

Comment:
User's should not use a `Passphrase` in order to log in. We only reqire an email address and password to authenticate the user currently.

Reply:
Aha. Wasn’t actually using it for auth, it was there from before. I’m removing it. 

(Just email and password are used for logging in)
https://github.com/universelabs/universe-webapp/blob/65c352f62bafc809f122d7fa96f2711461019300/client/src/components/LogInForm.js#L11-L13

Reply:
It was actually not there from before. But yes. Let’s remove it please. 
Thank you sir.

https://github.com/universelabs/universe-webapp/blob/8900d6c9d228189bcb2babbf7be0dc38abc13b47/client/src/components/LogInForm.js

Reply:
It seems I somehow picked it up from way back 😅
https://github.com/universelabs/universe-webapp/commit/d8fe10f813114c0f19927bbfad6fbbf8aa1b90ae#diff-92fc8f1f24c44cd5c75ca80953f8f5a0R59

Anw, it’s gone now 👍